packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx

```diff
@@ -526,9 +526,9 @@ const VideoGalleryStory = (args): JSX.Element => {
     .split(',')
     .map((p) => p.trim())
     .filter((p) => p)
-    .map((p, i) => {
+    .map((p) => {
       return {
-        userId: `user${i}`,
+        userId: `userId-${p}`,
```

Comment:
Needed to change the id because when I test this fix on storybook preview it did not work. When I remove pinned participant 'Daryl' with user id 'user2', someone else would then take user id 'user2' because we assign them to our test participants sequentially.